app/models/course.rb

```diff
@@ -84,16 +84,27 @@ class Course < ApplicationRecord
   # :revisions and :all_revisions have the same default implementation,
   # but a course type may override :revisions.
   has_many(:revisions, lambda do |course|
```

Comment:
Changing the base `revisions` scope seems like a riskier and more disruptive strategy to me. I think you probably want to go the opposite route and add `tracked_revisions` as a new scope that builds on top of revisions.

Reply:
Ideally, that would also avoid the need to change any existing specs. Ideally, this new feature wouldn't break any specs, since it will only change the behavior of the system in cases where there are ArticlesCourses that have been explicitly marked for exclusion.